src/linker/Linker.Analysis/README.md

```diff
@@ -0,0 +1,61 @@
+# Linker analyzer
+
+We have been prototyping a callgraph-based analyzer for the linker, that reports "linker-unsafe" usage in the IL reachable from some set of entry points (Main, or all public surface area). It simulates attributes that influence how unsafe patterns get reported.
+
+# Instructions
+
+To give it a try:
+
+## Clone and build
+```
+> git clone https://github.com/vitek-karas/linker -b Analysis
+> cd linker
+linker> ./eng/dotnet.{sh/ps1} build illink.sln
+```
+
+## Run
+
+Run the linker specifying the desired set of roots, optimizations, etc. A summary of the analysis will be written to stdout, and the "callstacks" will be written to `trimanalysis.json` in the linker output directory.
+For example: 
+
+```
+linker> dotnet ./src/ILLink.Tasks/bin/Debug/netcoreapp2.0/illink.dll -a console -c link -u link -d testapps/console/out/ --skip-unresolved true --ignore-descriptors true --verbose --dump-dependencies --used-attrs-only true -out testapps/console/linked --analyze-for-trim
```

Comment:
There's really no need for `--dump-dependencies` so we should probably leave it out.